lib/srv/db/server.go

```diff
@@ -829,7 +829,7 @@ func (s *Server) Close() error {
 
 // Shutdown performs a graceful shutdown.
 func (s *Server) Shutdown(ctx context.Context) (err error) {
-	err = s.close(ctx)
+	err = trace.Wrap(s.close(ctx))
```

Comment:
We should get rid of the named return while we're here.

https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters

> Naked returns are okay if the function is a handful of lines. Once it's a medium sized function, be explicit with your return values. Corollary: it's not worth it to name result parameters just because it enables you to use naked returns. Clarity of docs is always more important than saving a line or two in your function.

Reply:
heh, I did that but figured it would just clutter the diff more so I reverted. I'll remove since you prefer it too